src/pages/junghyowon/Login/JungLogin.scss

```
@@ -0,0 +1,119 @@
+@import url('https://fonts.googleapis.com/css2?family=Lobster&display=swap');
+
+.container-wrap {
+  width: 350px;
+  margin: 50px auto 10px auto;
+  border: 1px solid lightgray;
+  background-color: #fff;
+
+  h1 {
+    margin-top: 30px;
+    font-family: 'Lobster', sans-serif;
+    font-size: 40px;
+    line-height: 4rem;
+    text-align: center;
+  }
+
+  .input-container {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    width: 100%;
+
+    .input-wrap {
+      width: 270px;
+      margin: 35px auto 8px auto;
+    }
+
+    input,
+    textarea {
+      width: 270px;
+      height: 35px;
+      margin-bottom: 5px;
+      padding: 10px 12px;
+      border: 1px solid silver;
+      border-radius: 3px;
+      background-color: #f2f3f5;
+      color: #000000;
+      font-size: 12px;
+    }
+
+    input:focus {
+      outline: none;
+    }
```

JungLogin.js에 textatea태그를 사용 안하신거 같아서 삭제하셔도 될것같아요. textarea를 삭제하면 다음과 같이 쓸 수 있어요!
```suggestion
    input {
      width: 270px;
      height: 35px;
      margin-bottom: 5px;
      padding: 10px 12px;
      border: 1px solid silver;
      border-radius: 3px;
      background-color: #f2f3f5;
      color: #000000;
      font-size: 12px;
      &:focus {
        outline: none;
      }
    }
```